src/main/java/com/mcmiddleearth/mcmemusic/regionCheck/RegionCheck.java

```diff
@@ -41,10 +65,11 @@ public void containCheck(Player p, Polygonal2DRegion region, Region musicRegion)
     @Override
     public void run() {
         Bukkit.getOnlinePlayers().stream()
-              .filter(player -> player.hasPermission(Permission.LISTEN.getNode())
```

Comment:
Um you might have to add this back, I downloaded the main branch, so it didn't include some of the fixes